python/orca/test/bigdl/orca/data/test_spark_backend.py

```diff
@@ -112,16 +112,6 @@ def test_read_parquet(self):
 
         shutil.rmtree(temp)
 
```

Comment:
remove one empty line

Reply:
removed, see the latest commit: https://github.com/intel-analytics/BigDL/blob/05ce85d99e6652873da62bf3152e8a9d9a4c11b4/python/orca/test/bigdl/orca/data/test_spark_backend.py